docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
         - app.env
         - db.env
     # there is no `depend_on: condition:` in compose service v3
-    entrypoint: sh -c 'sleep 5; /app/entrypoint.sh'
+    entrypoint: sh -c 'sleep 60; /app/entrypoint.sh'
```

Comment:
Yeah I'm sorry for this quite ugly hack with bit more work where the app just waits for db socket to be available this wouldn't be needed anymore.